tests/core/lib/test_user_added_http_headers.py

```
@@ -0,0 +1,491 @@
+# tests/test_http.py
+import asyncio
+import types
```

Ruff (0.12.2)

`types` imported but unused

Remove unused import: `types`

(F401)